Sources/XiEditor/RPCSending.swift

```diff
@@ -76,7 +76,9 @@ class StdoutRPCSender: RPCSending {
         let errLogPath = errorLogDirectory?.path
         let errLogArgs = errLogPath.map { ["--log-dir", $0] }
         task.launchPath = path
-        task.arguments = errLogArgs
+        if let errLogArgs = errLogArgs {
```

Comment:
This is allowed to be nil, so this check isn't strictly necessary, although it doesn't do any harm.

Reply:
Ok, I didn't describe it enough 😄

Without this check we get into the following during assigning `task.arguments` a `nil` value:
> caught "NSInvalidArgumentException", "must provide array of arguments"
I added this check in order to avoid tests failure. I'm pretty sure it won't do us any harm 😄